Apps/g.Tec/g.NEEDaccess/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+cmake_minimum_required(VERSION 3.0)
+project(gNEEDaccess VERSION 0.0.1)
+
+# load LSLAppBoilerplate if not done already
+if(NOT TARGET LSL::lsl)
+    if(NOT LSL_ROOT)
+        set(LSL_ROOT "${CMAKE_CURRENT_LIST_DIR}/../../../build/lsl_Release/lslinstall/LSL")
+        get_filename_component(LSL_ROOT ${LSL_ROOT} ABSOLUTE)
+        if(NOT EXISTS ${LSL_ROOT})
+            message(FATAL_ERROR "When building a single app out of tree you need to set LSL_ROOT ( cmake -DLSL_ROOT=/path/to/installed/lsl)")
+        endif()
+    endif()
+    list(APPEND CMAKE_MODULE_PATH "${LSL_ROOT}/cmake")
+    include(LSLAppBoilerplate)
+endif()
+
+# GENERAL CONFIG #
+
+set(CMAKE_INSTALL_PREFIX ${LSL_ROOT}/..)
+
+LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+# set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
+
+# THIRD PARTY LIBRARIES #
+
+# Qt
+if(NOT Qt5_DIR)
```

Comment:
This will break as soon as Qt 5.9.2 comes out and then we'd need to change it everywhere.
We could have a cmake file in the root cmake directory with sane defaults that get included everywhere, so the defaults are in one place.

Reply:
Agreed, but what's the syntax for that to be called from here? add_subdirectory('../../../ etc)?

Reply:
With the `list(APPEND CMAKE_MODULE_PATH "${LSL_ROOT}/cmake")` above `include(LSLSaneDefaults)` should work out of the box, even with an installed tree (the cmake dir gets copied to the install location).